ruby/Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+group :development, :test do
+  gem 'minitest'
+end
```

Comment:
This isn't needed since `minitest` is a core gem that comes with Ruby, I believe. I've never needed to install it for testing our gem, neither locally nor in the GitHub action.